src/dev_server/index.tsx

```diff
@@ -102,14 +122,18 @@ const DevServer = props => {
     }
 
     connection.current = TcpSocket.createConnection({
-      port: config.remoteSocketPort,
+      port: 9092,
```

Comment:
Can we have this 9092 just be a hidden default? I feel like giving people control of their ports is a nice thing we could have in the future.